COMPONENTS/navigation/index.css

```diff
@@ -1,23 +1,123 @@
 @import url(/CSS/global.css);
 
-.navigation_comp {
+:root {
+  --header-height: 4rem;
+  --gray-light: 168, 168, 168;
+}
+
+.header {
+  width: 100%;
+  height: var(--header-height);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0rem 2rem;
+  background-color: white;
+  border-bottom: 1.5px solid rgba(var(--gray-light), 0.8);
+  overflow: hidden;
+  flex-wrap: nowrap;
+}
+
+.header_links {
+  position: absolute;
   display: flex;
-  justify-content: space-evenly;
-  background-color: rgba(var(--primary), 0.9);
-  color: white;
+  top: var(--header-height);
+  right: -100%;
+  justify-content: space-between;
+  align-items: flex-start;
+  transition: ease 500ms;
+  flex-direction: column;
+  background-color: rgb(184, 184, 184);
+  padding: 1.2rem 1.8rem;
+  gap: 16px;
```

Comment:
if you are using rem, have consistency don't change units in between. Use px for values less than 10px